tools/NuGet/template-artifactory/DynamoVisualProgramming.Tests.nuspec

```diff
@@ -14,18 +14,24 @@
         2) SystemTestServices.dll
         3) TestServices.dll</description>
         <summary>This package contains all that is required to get up and running creating tests for Dynamo libraries.</summary>
-        <copyright>Copyright Autodesk 2020</copyright>
+        <copyright>Copyright Autodesk 2023</copyright>
         <dependencies>
             <group targetFramework=".NETFramework4.8">
                 <dependency id="DynamoVisualProgramming.Core" version="$Version$"/>
             </group>
+            <group targetFramework="net6.0">
+                <dependency id="DynamoVisualProgramming.Core" version="$Version$"/>
+            </group>
         </dependencies>
     </metadata>
     <files>
         <!--These assemblies are referenced from the bin directory since they are not harvested for the installer-->
         <file src="..\..\..\bin\AnyCPU\Release\DynamoCoreTests.dll" target="lib\net48" />
         <file src="..\..\..\bin\AnyCPU\Release\SystemTestServices.dll" target="lib\net48" />
         <file src="..\..\..\bin\AnyCPU\Release\TestServices.dll" target="lib\net48" />
+        <file src="..\..\..\bin\NET60_Windows\Release\DynamoCoreTests.dll" target="lib\net6.0" />
+        <file src="..\..\..\bin\NET60_Windows\Release\SystemTestServices.dll" target="lib\net6.0" />
+        <file src="..\..\..\bin\NET60_Windows\Release\TestServices.dll" target="lib\net6.0" />
```

Comment:
I added this nuspec for tests for net6: https://github.com/DynamoDS/Dynamo/blob/master/tools/NuGet/template-net60-core-runtime/DynamoVisualProgramming.TestsNet6.nuspec

Reply:
@aparajit-pratap - in this PR I am trying to use a single nuspec for the both net6 and net48 as that is idiomatic. I don't think there will be an issue doing it this way, as these are quite small.

We'll use two nuspecs for the runtimecore - because its hundreds of megs.

Is that ok with you, so I will remove this separate nuspec?

Reply:
I guess that's fine by me, it's just a little confusing, that's all.

Reply:
Never mind my last comment, I suppose this was the original intent of this PR, which was to add net6 TFM to public nugets, I forgot.